### ISSUE TITLE
How to batch upload data to geoserver when geonode is running on docker

### ISSUE THREAD

**Issue by dakula009** [GeoNode/geonode#4112]
Hello, 

My Geonode is running in a docker container on a Ubuntu vm. I'd like to batch upload some data using geoserver's "import data" function. 

When browsing data, I realized that I'm not seeing "real" directories on the machine, instead, I'm actually browsing folders inside the container. So I cannot upload anything from local directories. That's why the there is nothing shown in the /home:

![image](https://user-images.githubusercontent.com/45241071/49675434-8adb5300-fa2a-11e8-818d-17e1fdcc18fc.png)

So my question is, how do I add data from my local directory to the geoserver that is running in a container? Things must be a lot easier when the geonode runs on your own machine, instead of docker.  I found this official tutorial,  but found it was really hard to follow: 

http://docs.geonode.org/en/master/tutorials/advanced/adv_data_mgmt/adding_data/creating_setting.html

Could someone kindly provide some guidance on this? Many thanks!

**Comment by t-book** [comment]
Hi @dakula009  I think I´ve already answered you at gitter, right? did this work for you?

**Comment by dakula009** [comment]
> Hi @dakula009 I think I´ve already answered you at gitter, right? did this work for you?

Hi @t-book  I just saw your message at gitter. Let me try it and get back to you!

Thanks!

**Comment by francbartoli** [comment]
@dakula009 can we close this?

**Comment by dakula009** [comment]
Yes please.

**Comment by afabiani** [comment]
@t-book @dakula009 it would be very helpful to report here the steps to fix the issue so that people looking for the same one in the future will be able to recognize the solution here.

Thanks,
-A.

**Comment by t-book** [comment]
The geoserver data volume is mounted to your django container under /geoserver_data/data.
 Means you can copy your layers just to it by use of 
`docker cp /HOSTPATH/YOURFOLDER/. b06b43f7d239:/geoserver_data/data/tmp/YOURFOLDER` (get the container id with `$ docker ps`). Now you should see the folder from geoserver’s import GUI. 

Finish the import by using updatelayers inside the container (log into with `docker exec -it b06b43f7d239 bash`)

```
source $HOME/.override_env; \
    echo DATABASE_URL=$DATABASE_URL; \
    echo GEODATABASE_URL=$GEODATABASE_URL; \
    echo SITEURL=$SITEURL; \
    echo ALLOWED_HOSTS=$ALLOWED_HOSTS; \
    echo GEOSERVER_PUBLIC_LOCATION=$GEOSERVER_PUBLIC_LOCATION; \
    django-admin updatelayers
```

**Comment by francbartoli** [comment]
At same point we can also add some shell script examples under a new folder `scripts/docker/misc` with the most useful geonode commands like `updatelayers`

**Comment by t-book** [comment]
... and update docs :smirk:

**Comment by lcoandrade** [comment]
Is there a way to make this whole process less cumbersome?
